tutormfe/patches/openedx-lms-production-settings

```diff
@@ -14,3 +14,31 @@ PROFILE_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https://{% else %}http://{% en
 LOGIN_REDIRECT_WHITELIST.append("{{ MFE_HOST }}")
 CORS_ORIGIN_WHITELIST.append("{% if ENABLE_HTTPS %}https://{% else %}http://{% endif %}{{ MFE_HOST }}")
 CSRF_TRUSTED_ORIGINS.append("{{ MFE_HOST }}")
+
+ALLOWED_HOSTS.append("{{ MFE_HOST }}")
```

Comment:
Why is this needed now if it wasn't needed before?

Reply:
I don't remember the exact reason, may be because now LMS would recevice a request from MFE_HOST as a proxy, might not be necessary but then would need to double test again

Reply:
I'm having a second look at this. I think that the "right" way to do this is not to modify ALLOWED_HOSTS, which is a sensitive setting, but to modify the Caddy configuration to automatically modify the "Host" header on forwarding to the LMS. I'll open a PR.

Reply:
Yes it can rewrite the headers, I was able to remove that line while changing the caddy patch to 

```caddy
    reverse_proxy /api/mfe_config/v1* lms:8000 {
        header_up Host {http.reverse_proxy.upstream.hostport}
    }
```

That did it for me.